entities/Rules/highestRollingPlayerGoesFirst.js

```diff
@@ -17,6 +18,11 @@ module.exports = function highestRollingPlayerGoesFirst({ UI }, { players }) {
     {}
   );
   // TODO: Handle Ties
-  let shiftValue = (players.length - highest) % players.length;
+  let shiftValue = highest;
```

Comment:
We can remove this line and rename `highest` to `shiftValue` in the destructuring on line 15